test/vanilla/legacy/Expected/AcceptanceTests/Constants/constants/_configuration.py

```diff
@@ -31,9 +31,6 @@ def __init__(
         # type: (...) -> None
         super(AutoRestSwaggerConstantServiceConfiguration, self).__init__(**kwargs)
 
-        self.header_constant = True
```

Comment:
I think you need to regenerate again, this information shouldn't be lost. I think what you might need to do is pull latest `autorestv3` branch, remove your `node_modules` folder in the root of `autorest.python`, then run `npm install` again.

The reason for this is I updated the testserver to add this test, and you installed the testserver before this change. Node modules don't automatically update, so you need to reinstall it to get this new testserver

Reply:
got it